<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilmQA_netOD_normalizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,1" columnminimumwidth="0,0">
    <item row="0" column="0">
     <widget class="MyGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1">
      <item>
       <widget class="ProcessStepWidget" name="processStepWidget_1" native="true"/>
      </item>
      <item>
       <widget class="ProcessStepWidget" name="processStepWidget_2" native="true"/>
      </item>
      <item>
       <widget class="ProcessStepWidget" name="processStepWidget_3" native="true"/>
      </item>
      <item>
       <widget class="ProcessStepWidget" name="processStepWidget_4" native="true"/>
      </item>
      <item>
       <widget class="ProcessStepWidget" name="processStepWidget_5" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShow_Document"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShow_Document">
   <property name="text">
    <string>Show Document</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>my_graphics_view.h</header>
  </customwidget>
  <customwidget>
   <class>ProcessStepWidget</class>
   <extends>QWidget</extends>
   <header>process_step_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
